.github/goreleaser.yaml

```diff
@@ -105,3 +105,18 @@ dockers:
       - "--label=org.opencontainers.image.source={{.GitURL}}"
       - "--label=repository=http://github.com/pomerium/pomerium"
       - "--label=homepage=http://www.pomerium.io"
+
+  - image_templates:
+      - "gcr.io/pomerium-io/pomerium:{{ .Tag }}-cloudrun"
```

Comment:
We are not publishing a latest tag from github actions.  (a) slightly less changes to maintain (b) cloudrun pins you to a commit hash of any tag anyway.  I think cosmetic tags are apt to create slightly more confusion in usage/upgrades.